cmd/versioninfo/BUILD

```diff
@@ -0,0 +1,17 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
We haven't added Bazel to this project. If we are looking to do that, I think we should do that in another PR

Reply:
Yep! Working on adding bazel in a separate branch: https://github.com/justaugustus/cluster-api-provider-azure/tree/build

I'll clean the bazel references up in this one.

Reply:
Removed!